sycl/test-e2e/Matrix/joint_matrix_tf32.cpp

```diff
@@ -10,7 +10,7 @@
 // RUN: %{build} -o %t.out -DSYCL_EXT_ONEAPI_MATRIX_VERSION=4
 // RUN: %{run} %t.out
 
-// XFAIL:*
+// XFAIL:CPU
```

Comment:
AFAIK, this check is case-sensitive
```suggestion
// XFAIL:cpu
```

Reply:
Thanks, done